README.md

```diff
@@ -85,7 +87,7 @@ most one such item per line. This allows the git notes to be automatically
 merged using the "cat\_sort\_uniq" strategy.
 
 Since these notes are not in a human-friendly form, all of the refs used to
-track them start with the prefix "refs/notes/devtools". This helps make it
+track them start with the prefix "refs/notes/git-review". This helps make it
```

Comment:
I can see arguments on both sides of this. Yes, we could use something like "refs/notes/git-appraise" to remove ambiguity as to which tool "owns" those refs, but the flip side of that ambiguity is inclusiveness.

I want other tools to feel encouraged to use the same formats, so that they all integrate in a loosely-coupled manner. I may be over thinking this, but "refs/notes/devtools" feels more welcoming to projects like GitLab, Gogs, or Jenkins.